src/components/concentration-display/Concentration.tsx

```diff
@@ -34,18 +40,33 @@ const Concentration: React.FC<AgentProps> = ({
     const { isPlaying, maxConcentration, page, getAgentColor } =
         useContext(SimulariumContext);
     const [width, setWidth] = useState<number>(0);
+    const [highlightState, setHighlightState] = useState<HighlightState>(
+        HighlightState.Initial
+    );
+
+    if (page === 8 && !isPlaying && highlightState === HighlightState.Initial) {
```

Comment:
Any way to make this not depend on a specific page number with the current content flow updates?

Reply:
no, unfortunately. this is just the page where we introduce the slider

Reply:
Gotcha. I left a similar note on the base branch's PR, but would love to see these replaced with a named constant...